systemtests/src/test/java/io/enmasse/systemtest/standard/TopicTest.java

```diff
@@ -237,6 +237,34 @@ void testMessageSelectorsProperty() throws Exception {
                 received.get(1, TimeUnit.MINUTES).get(0).getGroupId(), is(groupID));
     }
 
+    @Test
+    void testDurableSubscriptionOnPooledTopic() throws Exception {
+        Destination topic = Destination.topic("mytopic", "pooled-topic");
+        Destination subscription = Destination.topic("mysub", "standard-subscription");
```

Comment:
Shouldn't this be Destination.subscription?

Reply:
@lulf, yes! Thanks! Been chasing that down for ages. :(